flow-fixtures/test.js

```diff
@@ -15,4 +20,13 @@ test('Some content', async assert => {
   const simulator = getSimulator(app);
   const a: string = simulator.getService(TokenA);
   assert.equal(a, 'test');
+  createRequestContext('/');
+  createRenderContext('/');
+  createRequestContext('/', {
```

Comment:
Should we add a test to ensure that the options are passed through to the request correctly?

Reply:
Added tests